openbsd/Platform.c

```diff
@@ -369,14 +369,14 @@ void Platform_getBattery(double* percent, ACPresence* isOnAC) {
    *percent = NAN;
    if (found) {
       /* last full capacity */
-      mib[3] = 7;
+      mib[3] = SENSOR_WATTHOUR;
       mib[4] = 0;
       double last_full_capacity = 0;
       if (sysctl(mib, 5, &s, &slen, NULL, 0) != -1)
          last_full_capacity = s.value;
       if (last_full_capacity > 0) {
          /*  remaining capacity */
-         mib[3] = 7;
+         mib[3] = SENSOR_WATTHOUR;
          mib[4] = 3;
```

Comment:
Note: `Remaining capacity reading`

Reply:
Do you suggest me to change this number to 6, or should I just keep the current number (3) for now?

Reply:
Edit: Ignore my previous message. I can tell why the number is 3 now instead of 6.